Modules/TPC/src/PID.cxx

```diff
@@ -69,6 +69,11 @@ void PID::initialize(o2::framework::InitContext& /*ctx*/)
       getObjectsManager()->startPublishing(hist.get());
     }
   }
+  for (auto const& pair : mQCPID.getMapOfCanvas()) {
+    for (auto& canv : pair.second) {
+      getObjectsManager()->startPublishing<true>(canv.get());
+    }
+  }
```

Comment:
why publish those objects if they were not published before?

Reply:
It must have been changed since I did the PR, good catch